include/boost/geometry/util/precise_math.hpp

```diff
@@ -347,29 +355,51 @@ inline RealNumber orient2dtail(vec2d<RealNumber> const& p1,
     return(D[D_nz - 1]);
 }
 
-// see page 38, Figure 21 for the calculations, notation follows the notation in the figure.
+// see page 38, Figure 21 for the calculations, notation follows the notation
+// in the figure.
 template
 <
     typename RealNumber,
-    std::size_t Robustness = 3
+    std::size_t Robustness = 3,
+    typename EpsPolicy
 >
 inline RealNumber orient2d(vec2d<RealNumber> const& p1,
                            vec2d<RealNumber> const& p2,
-                           vec2d<RealNumber> const& p3)
+                           vec2d<RealNumber> const& p3,
+                           EpsPolicy& eps_policy)
 {
-    if(Robustness == 0)
-    {
-        return (p1.x - p3.x) * (p2.y - p3.y) - (p1.y - p3.y) * (p2.x - p3.x);
-    }
+    auto const x = p3.x;
+    auto const y = p3.y;
+
+    auto const sx1 = p1.x;
+    auto const sy1 = p1.y;
+    auto const sx2 = p2.x;
+    auto const sy2 = p2.y;
```

Comment:
This was done to improve readability correct?
If that's the case and `p1`, `p2` and `p3` have some specific meaning then maybe it's these variables that should be renamed?

Reply:
I don't understand why aren't they used below too. E.g. below there is `t1[0] = p1.x - p3.x;` which would translate to `t1[0] = sx1 - x;` or `t1[0] = -dpx;`. Does these make sense? If they doesn't it means that the names are incorrect.

Reply:
I followed the naming of `side_by_triangle` but that complicated things. This can be simplified. Actually the `eps_policy` can be created/updated using `t1,...,t4`. See https://github.com/boostorg/geometry/pull/873/commits/e3eb4bae2cbac767ca1ff106dcbce504fcd0771c